Documentation/Examples/PdfaShowCase/Index.rst

```diff
@@ -0,0 +1,218 @@
+﻿.. ==================================================
+.. FOR YOUR INFORMATION
+.. --------------------------------------------------
+.. -*- coding: utf-8 -*- with BOM.
+
+.. include:: ../../Includes.txt
+
+
+.. _pdfashowcase:
+
+PDF/a Show Case
+======================
+
+
+.. _pdfashowcase_intro:
+
+Intro
+-----
+
+Its's possible to render the output as a valide PDF/a document.
+
+.. _pdfashowcase_typoscript:
+
+TypoScript
+----------
+
+Be sure to disable all header data, to ensure taht all HTML header output is disabled. Some variables on page properties are used to fill the meta data.
+
+::
+
+	pdfpage = PAGE
+	pdfpage {
+
+		config {
+			disableAllHeaderCode = 1
+			xhtml_cleaning = 0
+			admPanel = 0
+		}
+
+		10 = FLUIDTEMPLATE
+		10 {
+			file = EXT:pdfviewhelpers/Resources/Public/Examples/PdfaShowCase/Template.html
+            variables {
```

Comment:
@koritnik could you fix the indentation here? .rst files are sensitive to correct indentation